### ISSUE TITLE
New Page: List of all Students

### ISSUE THREAD

**Issue by garg3133** [garg3133/JagratiWebApp#32]
We need a new page to be displayed at `/students/` to list all the students currently a part of the initiative, in the form of clickable cards which will take us to the student profile upon click.

To Do:
- [ ] Create an empty HTML page displayed at `/students/` endpoint.
- [ ] Fetch student data from database and display in form of cards.
- [ ] Add filter to filter the students section-wise.

Working on this issue will require knowledge of HTML, CSS + Bootstrap4 and a basic knowledge of Django.

For any further doubts and queries, feel free to join the community and ask there: https://discord.gg/Ek9q45ZjAv

**Comment by garg3133** [comment]
@prakhar1144 You can work on this issue if you like to!

**Comment by prakhar1144** [comment]
@garg3133  Sure, please assign it to me.

**Comment by 07yuvraj** [comment]
hey @garg3133 i want to work on this.May i?

**Comment by garg3133** [comment]
Sure @07yuvraj, you can work on this issue if @prakhar1144 is not.

@prakhar1144 Can you please respond if you are working on this or not?

**Comment by 07yuvraj** [comment]
@garg3133 so may i start working on it or wait for him?

**Comment by prakhar1144** [comment]
@07yuvraj you can work on it.

**Comment by 07yuvraj** [comment]
![Screenshot (85)](https://user-images.githubusercontent.com/55285282/103155299-7068a380-47c4-11eb-8298-879e3f193c49.png)
i am getting this error

**Comment by garg3133** [comment]
Did you run `pip install -r requirements.txt`.

**Comment by 07yuvraj** [comment]
yeah i have done that but on installing pillow there were some errors

**Comment by garg3133** [comment]
Then I think other requirements are also not installed properly. Try googling the error you are getting while installing pillow.

**Comment by harikishantk** [comment]
I have met the requirement which you have specified above, can I work on it? If yes, kindly assign me.

**Comment by garg3133** [comment]
Yes @harikishantk, you can work on this.

**Comment by N00Bmaster2810** [comment]
@garg3133 sir, if this issue is not resolved then please can you assign this to me?

**Comment by garg3133** [comment]
Hey @harikishantk, are you still working on this issue?

**Comment by harikishantk** [comment]
Already I'm working on modal issue, I would like to work on this issue later, if there is urgency, you can assign someone as well. My apologies.

**Comment by harikishantk** [comment]
Yep

**Comment by tmo66** [comment]
Hey, @harikishantk are you still working on this?

**Comment by harikishantk** [comment]
Yes I am, under the GSSoC 21' .

**Comment by sakshi21-03** [comment]
@garg3133 , i would like to work on this part.

**Comment by garg3133** [comment]
Hey @sakshi21-03, @harikishantk is currently working on this issue. Would you like to work on something else? Like we need a new page to display all the feedbacks received in `feedbacks` app. Would you like to work on that?

**Comment by sakshi21-03** [comment]
> Hey @sakshi21-03, @harikishantk is currently working on this issue. Would you like to work on something else? Like we need a new page to display all the feedbacks received in `feedbacks` app. Would you like to work on that?

yes @garg3133 , i  would like to work on this part

**Comment by Haneesha123** [comment]
@garg3133 I would like to work on this issue if no one else is.

**Comment by harikishantk** [comment]
Hey @Haneesha123 , I'm already working on it.

**Comment by Ananya9878** [comment]
Pls assign this issue to me

**Comment by harikishantk** [comment]
Hey @Ananya9878 , I'm already working on it. Please check other issues.

**Comment by harikishantk** [comment]
Yes I'm working on it. I gave front end update on it.
